app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,2 @@
+@import 'bootstrap/scss/bootstrap';
```

Comment:
This seems to be the source of the second error.

Reply:
Thank you for the feedback but that is an error on your side. The project was reviewed twice before this and no one raised this issue.